From 00ad193c34bf13985df8967ebfa48aed9ebeb023 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Mon, 7 Mar 2016 17:21:52 +0200
Subject: [PATCH 0114/1345] mvebu: Select gic-v2m for ARCH_MVEBU

commit  17861e71bb9535d861e3d3895563eaa7665fc5ac from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This is needed by some of MVEBU drivers for interrupt signalling -
xor-v2, ppv2.1...

Change-Id: Ib4daefaf3c9d05ea8f1e68a67838a61621b92b99
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28091
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/Kconfig.platforms |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 4116191..cc6776a 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -111,6 +111,7 @@ config ARCH_MVEBU
 	select MVEBU_ODMI
 	select MVEBU_CLK_COMMON
 	select ARMADA_3700_CLK
+	select ARM_GIC_V2M
 	select MVEBU_PIC
 	select OF_GPIO
 	select ARMADA_CP110_CLK
-- 
1.7.9.5

